Dockerfile

```diff
@@ -1,15 +1,19 @@
 FROM behance/docker-nginx:1.1.1
 MAINTAINER Bryan Latten <latten@adobe.com>
 
-# Install singularity_runner
-RUN apt-get install build-essential ruby1.9.1-dev -y && \
```

Comment:
note: singularity *no longer* an internal container dependency. only operates from the outside